CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Features
+- Updated users addresses 'default' endpoint to have the correct path ([#148](https://github.com/whirli/whirli-client/pull/148)).
```

Comment:
Remember to write using imperative verbs. See the other changelog entries for info ("update" not "updated").

This also sounds like a bugfix, if it previously had the incorrect path. If that's right, could you amend the title to `### 🐛 Bug Fixes` please